.github/workflows/python-tests.yml

```diff
@@ -22,3 +30,8 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
@ilyastrodubtsev can you change this to run a regular full linting on the code base? If we need to exclude something for now, use the `noqa` annotation.